helm-chart/amalthea/crd-spec.yaml

```diff
@@ -216,7 +216,7 @@ spec:
               type: string
           type: object
         size:
-          default: 100Mi
+          default: null
```

Comment:
I'd leave the default for now and just set `null` explicitly when needed, for example in renku-notebooks. We can start collecting a list of things that we want to change when we update the CRD to `v1beta1` or `v1alpha2` (and I agree that changing this default could be one of them), but I think it would be nice to defer this for a bit longer and keep the CRD stable.

Reply:
```suggestion
          default: 100Mi
```